recipes/default.rb

```diff
@@ -74,7 +74,8 @@
 # Else, if cuda/accept_nvidia_download_terms is set to true, then make 'gpu' true.
 if node['hops']['gpu'].eql?("false")
   if node.attribute?("cuda") && node['cuda'].attribute?("accept_nvidia_download_terms") && node['cuda']['accept_nvidia_download_terms'].eql?("true")
-     node.override['hops']['gpu'] = "true"
+    node.override['hops']['gpu'] = "true"
+    node.override['hops']['capacity']['resource_calculator_class'] = "org.apache.hadoop.yarn.util.resource.DominantResourceCalculatorGPU"      
```

Comment:
@jimdowling no need to overrride the `resource_calculator_class` here as `hops::default` includes `hops::_config`.

Reply:
ok